packages/shared/lib/core/wallet/actions/buildWalletState.ts

```diff
@@ -30,17 +31,21 @@ export async function buildWalletState(
         delegations: [],
     }
 
-    const depositAddress = ''
-    const votingPower = ''
+    let depositAddress = ''
+    let votingPower = ''
+    let accountsOutput: OutputData[] = []
+    let implicitAccountsOutput: OutputData[] = []
 
+    try {
+        balances = await wallet.getBalance()
+        depositAddress = await getDepositAddress(wallet)
```

Comment:
🔈    @cpl121 
`packages/shared/lib/core/wallet/utils/getDepositAddress.ts`
Note for the future: the implementation of `getDepositAddress` is incorrect, the deposit address should be the address of the selected block issuer account output , not the ed25519 address